src/App.js

```diff
@@ -25,6 +25,16 @@ function App() {
 
   return (
     <Router>
+      <head>
```

Comment:
Linkfree is currently using primereact components with tailwindcss for styling. Adding bootstrap would create some redundancy. Perhaps you could implement the same with the Accordion component provided by primereact: https://www.primefaces.org/primereact/accordion/ ?